sdk/keyvault/azkeys/CHANGELOG.md

```diff
@@ -1,18 +1,12 @@
 # Release History
 
-## 0.4.1 (Unreleased)
-
-### Features Added
+## 0.5.0 (2022-04-05)
```

Comment:
Shouldn't requiring 1.18 be a breaking change?

Reply:
Our convention for v0.x.y is to increment x for breaking changes, y for everything else.